From 7bed87eec383a60dc0e32c5ada07a8a051761dab Mon Sep 17 00:00:00 2001
From: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
Date: Fri, 9 Nov 2018 11:46:16 -0800
Subject: [PATCH 0750/1051] misc: otx_bphy_ctr: Increase the maximum BPHY
 interrupts to be supported

This patch increases the maximum BPHY interrupts that can be supported.

Signed-off-by: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/misc/otx_bphy_ctr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/otx_bphy_ctr.c b/drivers/misc/otx_bphy_ctr.c
index b0fcf4c6c965..d7d06118d5a0 100644
--- a/drivers/misc/otx_bphy_ctr.c
+++ b/drivers/misc/otx_bphy_ctr.c
@@ -20,7 +20,7 @@
 
 #define DEVICE_NAME	"otx-bphy-ctr"
 #define OTX_IOC_MAGIC	0xF3
-#define MAX_IRQ		4
+#define MAX_IRQ		23
 
 static struct device *otx_device;
 static struct class *otx_class;
-- 
2.17.1

